docs/Installation.rst

```diff
@@ -49,7 +49,12 @@ Installation
 
 8. Create your first course! If you want, give it a demo account too::
 
-   $ python dbmaint.py --create_course "Demo Course Name" --admin_email your_admin_email@company.com --admin_name "Your Name" [--create_demo]
+   $ python dbmaint.py --create_course "Demo Course Name" --admin_email your_admin_email@company.com --admin_name "Your Name" [--demo_email your_demo_email@company.com]
    course_key: leact-skio-proih
 
 9. You now have a course key! If the demo account is made, you have that too.
+
+10. To run a Plant-Tracer/webapp server process locally, examine the debug-* targets in Makefile. The general form is::
+
+    $ PLANTTRACER_CREDENTIALS=${MY_INI_FILES}/credentials-myconfig.ini python bottle_app.py [arguments]
```

Comment:
Strictly speaking, you don't need the `{` and `}`. Do you think that it makes it easier to read?

Reply:
I always use curly braces around shell variable references. Because sometimes you need them (not here, true) to prevent bad stuff from happening. They don't hurt, and by always using them, I don't have to remember the more complicated rules about when I need them.

I'd been taught that always using them was a sort of good style guideline, for consistency if nothing else, but it's not a hill I need to die on.